components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/listeners/UserManagementV2AuditLogger.java

```diff
@@ -216,10 +216,13 @@ public boolean doPostGetUserClaimValueWithID(String userId, String claim, List<S
             dataObject.put(CLAIM_VALUE_FIELD, new JSONArray(claimValue));
         }
         dataObject.put(PROFILE_FIELD, profileName);
-        AuditLog.AuditLogBuilder auditLogBuilder = new AuditLog.AuditLogBuilder(getInitiatorId(),
-                LoggerUtils.getInitiatorType(getInitiatorId()), userId, LoggerUtils.Target.User.name(),
-                GET_USER_CLAIM_VALUE_ACTION).data(jsonObjectToMap(dataObject));
-        triggerAuditLogEvent(auditLogBuilder);
+        String initiatorId = getInitiatorId();
+        if (!initiatorId.equals(LoggerUtils.Initiator.System.name())) {
```

Comment:
Shall we revert this other way? `LoggerUtils.Initiator.System.name().equalsIgnoreCase(initiatorId)`